tests/integration/targets/route53_health_check/tasks/create_delete_by_name.yml

```diff
@@ -0,0 +1,116 @@
+---
+- block:
+
+    - name: 'Create HTTP health check with use_unique_names - check_mode'
+      amazon.aws.route53_health_check:
+        state: present
+        name: '{{ tiny_prefix }}-{{ resource_path }}-test-hc-delete-if-found'
+        ip_address: '{{ ip_address }}'
+        port: '{{ port }}'
+        type: '{{ type_http }}'
+        resource_path: '{{ resource_path }}'
+        use_unique_names: true
+        tags:
+          Service: my-service
+          Owner: my-test-xyz
+          Lifecycle: dev
+        fqdn: '{{ fqdn }}'
+      check_mode: true
+      register: create_result
+
+    - name: 'Check result - Create HTTP health check'
+      assert:
+        that:
+        - create_result is not failed
+        - create_result is changed
+
+    - name: 'Create HTTP health check with use_unique_names'
+      amazon.aws.route53_health_check:
+        state: present
+        name: '{{ tiny_prefix }}-{{ resource_path }}-test-hc-delete-if-found'
+        ip_address: '{{ ip_address }}'
+        port: '{{ port }}'
+        type: '{{ type_http }}'
+        resource_path: '{{ resource_path }}'
+        use_unique_names: true
+        tags:
+          Service: my-service
+          Owner: my-test-xyz
+          Lifecycle: dev
+        fqdn: '{{ fqdn }}'
+      register: create_result
+
+    - name: Get ID's for health_check created in above task
+      set_fact:
+        health_check_1_id: "{{ create_result.health_check.id }}"
+
+    - name: 'Check result - Create HTTP health check'
+      assert:
+        that:
+        - create_result is not failed
+        - create_result is changed
```

Comment:
Can we add an assertion to check if Name tag is created?

Reply:
Opened PR to get required policy added https://github.com/mattclay/aws-terminator/pull/243